torchvision/csrc/models/modelsimpl.h

```diff
@@ -36,8 +36,8 @@ inline bool double_compare(double a, double b) {
 
 inline void deprecation_warning() {
   TORCH_WARN_ONCE(
-      "The vision::models namespace is deprecated since 0.12 and will be "
-      "removed in 0.14. We recommend using Torch Script instead: "
+      "The vision::models namespace is not actively maintained, use at "
+      "your own discretion. We recommend using Torch Script instead: "
```

Comment:
Looks like we originally didn't intend to deprecate the C++ models: at least according to https://github.com/pytorch/vision/pull/4375#pullrequestreview-751524747, the original intention was to stop maintaininng them but to still keep them around.

So I put back the original warning, instead of removing the files. I'll still wait for @datumbox 's input on this tho.

Reply:
Thanks for checking @NicolasHug. The intention was to remove them. The majority of the new models can't be added on C++ because the layers are only available on Python. So I think we are good to drop them, which was the intention when we deprecated them.